### ISSUE TITLE
Toevoegen DMS link en check  op inschrijfformulier

### ISSUE THREAD

**Issue by indiePeeters** [esac-ic/esac.nl#198]
Nieuwe leden dienen zich in te schrijven op het dms systeem van het sportcenrum voordat ze zich inschrijven voor ESAC. 

Dit moeten ze doen door op een link te drukken. Momenteel staat deze link op de lid worden pagina. Helaas kunnen we niet automatisch checken of dit daadwerkelijk gedaan is. 

Het is makkelijker dat de link in het inschrijfformulier toegevoegd wordt, dat er een checkbox die de gebruiker moet aanvinken toegevoegd wordt, En indien mogelijk een check gedaan wordt of er op de link geklinkt is.

**Comment by indiePeeters** [comment]
Dus voor de duidelijkheid,

1. Link toevoegen aan formulier
2. Checkbox met "Heb je je ingeschreven op het DMS systeem?" toevoegen
3. Check of er op de link geklikt is (ik weet niet of dit handig is?)

**Comment by otherwise777** [comment]
Is dit de link vanuit het SCC? https://dms.studentensportcentrumeindhoven.nl/nl/home
Hoe weet de ESAC of iemand is ingeschreven bij het SCC? gebeurt dit handmatig?

Ik zie dat de inlog gaat via een portal van de onderwijs instelling, is de eis dat ze zich eerst moeten aanmelden in het DMS systeem maar niet meteen een sportkaart te hoeven kopen?

**Comment by wouterbles** [comment]
> En indien mogelijk een check gedaan wordt of er op de link geklinkt is.

Dit zou ik niet doen, een checkbox lijkt me voldoende.

**Comment by indiePeeters** [comment]
> > En indien mogelijk een check gedaan wordt of er op de link geklinkt is.
> 
> Dit zou ik niet doen, een checkbox lijkt me voldoende. Niet heel gebruiksvriendelijk denk ik.

Lijkt me prima, Inderdaad stel een gebruiker heeft het al een keer gedaan dan zal het verwarrend zijn.

**Comment by indiePeeters** [comment]
> Is dit de link vanuit het SCC? https://dms.studentensportcentrumeindhoven.nl/nl/home
> Hoe weet de ESAC of iemand is ingeschreven bij het SCC? gebeurt dit handmatig?
> 
> Ik zie dat de inlog gaat via een portal van de onderwijs instelling, is de eis dat ze zich eerst moeten aanmelden in het DMS systeem maar niet meteen een sportkaart te hoeven kopen?

Yes dat is de link. Je logt inderdaad in via de onderwijsinstelling. Als het goed is moeten mensen daar eerst een sportkaart aanschaffen voordat ze zich in het DMS kunnen inschrijven voor de vereniging. Als dit gedaan is krijgt de secretaris een mailtje. Zodat we bij de inschrijving weten wie er een sportkaart heeft en wie niet.

**Comment by indiePeeters** [comment]
Het lijkt me trouwens handig als dit in een aparte card komt te staan waarin de volgende informatie en een checkbox staat.

1. Login met je Fontys of TUe account via de volgende link
2. Bij "Inschrijven Vereniging" selecteer "Alpinism - E.S.A.C."
3. Druk op "Bewaren"
